estela-web/src/shared/header/index.tsx

```diff
@@ -25,12 +25,32 @@ export class CustomHeader extends Component<unknown> {
 
     render(): JSX.Element {
         return (
-            <Header className="header">
-                <Link to="/">
-                    <div className="logo-header">estela</div>
-                </Link>
-                <Row justify="end">
-                    {this.isLogged() ? (
+            <Header className="bg-white">
+                <Row justify="center" align="middle">
+                    <Col flex={1}>
+                        <Link to="/">
+                            <div className="text-xl ">estela</div>
+                        </Link>
+                    </Col>
+                    {/* <Row justify="center"> */}
+                    <Col>
+                        <img src="notification.svg" width="26" className="mr-10" alt="" />
+                    </Col>
+                    <Col className="flex text-[#4D47C3]">
```

Comment:
@raymond1242 if you have been doing this in several other places, even in already merged PRs, please fix that. This is not the way to determine "colors" for the entire site.

Colors and important styles should be determined in a global css file, and the respective tags should be adding those classnames